ilastik/applets/tracking/base/trackingBaseDataExportApplet.py

```diff
@@ -185,4 +216,75 @@ def _configure_operator_with_parsed_args(cls, parsed_args, opTrackingDataExport)
         # configure super operator
         DataExportApplet._configure_operator_with_parsed_args(parsed_args, opTrackingDataExport)
 
+    def _export_with_plugin(self, lane_index: int, checkOverwriteFiles: bool, pluginName: str) -> bool:
+        argsSlot = self.topLevelOperator.AdditionalPluginArguments
+        pluginInfo = pluginManager.getPluginByName(pluginName, category="TrackingExportFormats")
+
+        if pluginInfo is None:
+            logger.error("Could not find selected plugin %s", pluginName)
+            return False
+
+        plugin = pluginInfo.plugin_object
+        logger.info("Exporting tracking result using %s", pluginName)
+
+        name_format = self.topLevelOperator.getLane(lane_index).OutputFilenameFormat.value
+        partially_formatted_name = self.getPartiallyFormattedName(lane_index, name_format)
+
+        if plugin.exportsToFile:
```

Comment:
```python
filename = partially_formatted_name
if plugin.exportsToFile and not os.path.basename(filename):
    filename = os.path.join(filename, 'pluginExport.txt')
```

OTOH, do we want to use the default file name if we get a directory name? If so, then we'd better to check whether `os.path.isdir(filename)` instead of hoping that the name we get ends with `/`.

Reply:
Regarding `os.path.isdir`
This needs a bit more work, not only in this code path. We need to handle user provided paths consistently.